amplify/backend/types/amplify-dependent-resources-ref.d.ts

```diff
@@ -0,0 +1,9 @@
+export type AmplifyDependentResourcesAttributes = {
```

Comment:
Type definitions should not be in the repo

Reply:
As I said, this PR isn't meant to be merged, it is just a basic to help developpers load the project on their machine. We need the actual schema, or an alternative method of storage.